core/src/test/java/org/apache/accumulo/core/file/BloomFilterLayerLookupTest.java

```diff
@@ -84,7 +80,9 @@ public void test() throws IOException {
 
     // get output file name
     String suffix = FileOperations.getNewFileExtension(acuconf);
-    String fname = new File(tempDir.getRoot(), testName + "." + suffix).getAbsolutePath();
+    String fname = new File(tempDir,
+        testInfo.getTestMethod().orElseThrow(IllegalStateException::new).getName() + "." + suffix)
```

Comment:
Already throws NoSuchElementException on .get if empty:
```suggestion
    String fname = new File(tempDir, testInfo.getTestMethod().get().getName() + "." + suffix)
```